wp-cli-templates/block-index-js.mustache

```diff
@@ -20,23 +21,16 @@ export const name = '{{ slug }}';
 export const title = __( '{{ title }}', 'jetpack' );
 export const settings = {
 	title,
-
 	description: (
 		<Fragment>
 			<p>{ __( '{{ description }}', 'jetpack' ) }</p>
 			<ExternalLink href="#">{ __( 'Learn more about {{ title }}', 'jetpack' ) }</ExternalLink>
 		</Fragment>
 	),
-
 	/* @TODO Add the icon. You can use one of these https://material.io/tools/icons/?style=outline */
```

Comment:
Should this `@TODO` be copied to `block-icon-js.mustache`, since it's more relevant there?